README.md

```diff
@@ -160,14 +160,14 @@ units_region = DynamicTableRegion(
     data=region_indices, table=units_table, description="region of units table", name="units_region"
 )
 
-event_timestamps = np.array([0.25, 5.0, 12.25])
+timestamps = np.array([0.25, 5.0, 12.25])
 milliseconds_from_event_to_first_bin = -50.0  # The first bin is 50 ms before the event
 bin_width_in_milliseconds = 100.0
 name = "BinnedAignedSpikesForMyPurpose"
 description = "Spike counts that is binned and aligned to events."
 binned_aligned_spikes = BinnedAlignedSpikes(
     data=data,
-    event_timestamps=event_timestamps,
+    timestamps=timestamps,
     bin_width_in_milliseconds=bin_width_in_milliseconds,
     milliseconds_from_event_to_first_bin=milliseconds_from_event_to_first_bin,
     description=description,
```

Comment:
this does not have event conditions

Reply:
Yeah, I introduce the possibility of storing multiple conditions in the next section. I think it makes presentation easier if less concepts are introduced at the same time.